### DISCUSSION TITLE
Consulta de la clase del 30/9/23

### DISCUSSION THREAD

**Discussion by jeremiasFox** [INGCOM-UNRN-P1/cursada-2023#214] [Preguntas y respuestas] [answered]
donde puedo encontrar la grabación del lunes 30, que nevó mucho gracias

**Comment by martinvilu** [comment, marked as answer]
No vimos nada en particular, estuvimos viendo cuestiones de la práctica. No tengo la grabación a mano, la misma será cargada apenas esté disponible.

**Comment by jeremiasFox** [comment]
a bueno gracias!
